libs/common/tsconfig.json

```diff
@@ -28,5 +28,9 @@
     /* Completeness */
     "skipLibCheck": true
   },
-  "include": ["**/*.ts"]
+  "ts-node": {
+    "files": true
+  },
+  "include": ["**/*.ts"],
+  "exclude": []
```

Comment:
Do we want to explicitly ignore `node_modules` as in other places?

Reply:
I'm not sure...I think for consistency we probably should, so I'll add that change and see if everything still works